src/odoo_sort_manifest_depends/sort_manifest_deps.py

```diff
@@ -7,11 +7,17 @@
 
 import click
 from click import command, option
+from diskcache import Cache
 from manifestoo_core.addon import Addon, is_addon_dir
 from manifestoo_core.core_addons import is_core_ce_addon, is_core_ee_addon
 from manifestoo_core.odoo_series import OdooSeries
+from requests import head
 
 NAME_DEFAULT_CATEGORY = "Default"
+OCA_ADDON_URL = "https://wheelhouse.odoo-community.org/oca-simple/odoo-addon-{addon_name}/"
```

Comment:
```suggestion
OCA_INDEX_URL= "https://wheelhouse.odoo-community.org/oca-simple/"
```

The use manifestoo-core `addon_name_to_distribution_name`

Reply:
I think that I spotted a small bug in the function `addon_name_to_distribution_name` : https://github.com/acsone/manifestoo-core/pull/78